### PR TITLE
RDCC-529 updated ErrorConstantsTest For QA PRD queries

### PR DESCRIPTION
**Pull request by jamesohmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-529

### Change description ###

Updated ErrorConstantTest for QA testing

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationServiceImplTest.java [hunk 1]
```diff
@@ -232,7 +230,6 @@ public void testSavesAnOrganisation() {
         prdEnums.add(new PrdEnum(new PrdEnumId(0, "SIDAM_ROLE"), "pui-user-manager", "SIDAM_ROLE"));
         prdEnums.add(new PrdEnum(new PrdEnumId(4, "ADMIN_ROLE"), "organisation-admin", "ADMIN_ROLE"));
         prdEnums.add(new PrdEnum(new PrdEnumId(10, "JURISD_ID"), "PROBATE", "PROBATE"));
-        prdEnums.add(new PrdEnum(new PrdEnumId(10, "JURISD_ID"), "PROBATE", "PROBATE"));
```
Comment (AdilOoze) [inline src/test/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationServiceImplTest.java]: Why was this removed? I believe this was required to satisfy a mutation scenario...

File: src/test/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationServiceImplTest.java [hunk 2]
```diff
@@ -598,23 +595,4 @@ public void testAllAttributesAddedToSuperUser() {
...
-        List<UserAttribute> attributes = new ArrayList<>();
-        attributes.add(userAttributeMock);
-        when(prdEnumRepositoryMock.findAll()).thenReturn(prdEnums);
-        when(prdEnumServiceMock.findAllPrdEnums()).thenReturn(prdEnums);
-        when(userAttributeRepositoryMock.saveAll(any())).thenReturn(attributes);
-
-        organisationServiceImplMock.createOrganisationFrom(organisationCreationRequest);
-
-        verify(userAttributeRepositoryMock, times(0)).saveAll(any());
-    }
```
Comment (AdilOoze) [inline src/test/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationServiceImplTest.java]: Again, I believe this was test was created to fulfil a mutation scenario

Comment (jamesohmcts) [reply src/test/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationServiceImplTest.java]: Hi this was an error and will be added back

File: src/test/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationServiceImplTest.java (hunk 1 above)
Comment (jamesohmcts) [reply src/test/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationServiceImplTest.java]: I believe that this line is a duplication of the line above

Comment (AdilOoze) [reply src/test/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationServiceImplTest.java]: 😂I come across like a goon, but ah cool, git didn't make that look very clear

File: src/test/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationServiceImplTest.java (hunk 2 above)
Comment (AdilOoze) [reply src/test/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationServiceImplTest.java]: nice 😄

File: src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/UserAttributeServiceImpl.java [hunk 3]
```diff
@@ -59,4 +60,33 @@ public void addUserAttributesToUser(ProfessionalUser newUser, List<String> userR
...
+        if (!CollectionUtils.isEmpty(attributes)) {
+
+            userAttributeRepository.saveAll(attributes);
+        }
+
+        return attributes;
+
+    }
+
+    private boolean isValidEnumType(String enumType,List<String> jurisdictionIds, PrdEnum prdEnum) {
```
Comment (akiohmcts) [inline src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/UserAttributeServiceImpl.java]: I know we worked on this together, but I didn't see it at the time. This conditional method can be simplified down to: PrdEnumType.values().containes(enumType) and put directly in the filter statement.
